### PR TITLE
Added Caesar spec

### PR DESCRIPTION
**Pull request by WeeknightMVP**
I don't see a Cryptol spec for Caesar ciphers among [Cryptol](https://github.com/GaloisInc/cryptol), [SAW](https://github.com/GaloisInc/saw-script), or [cryptol-specs](https://github.com/GaloisInc/cryptol-specs) repos (except in Chapter 2 of the [Programming Cryptol](https://github.com/GaloisInc/cryptol/blob/master/docs/ProgrammingCryptol/classic/Classic.tex) text), so here is one with some simple test vectors and properties.

### CODE REVIEW

File: specs/Caesar.cry [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/* Simple Caesar cipher (ROT13) */
+module Caesar where
```
Comment (weaversa) [inline specs/Caesar.cry:2]: I imagine `module specs::Caesar` would be better, just in case some ancillary files elsewhere (such as examples in the labs) want to reference modules from here.

File: README.md [hunk 2]
```diff
@@ -1,45 +1,48 @@
 # Programming with Cryptol and SAW
```
Comment (weaversa) [inline README.md:1]: Will you revert these changes to the README.md since you already have a pull request open for them?

File: specs/Caesar.cry (hunk 1 above)
Comment (WeeknightMVP) [reply specs/Caesar.cry]: Hmm...that would be amusing: a module collecting all the tutorial ciphers.  I'll start by mirroring the organization of [cryptol-specs](https://github.com/GaloisInc/cryptol-specs), under which Caesar is a symmetric block cipher...

**Comment by weaversa** [comment]
@WeeknightMVP I polished things a tad. Feel free to revert if you don't like it.
